docs/dashboards/your-first-dashboard.md

```diff
@@ -85,4 +85,4 @@ Thus we have to import the main dashboard package, the Highcharts library and a
         }]
     ```
 
-6. With that config, you dashboard should look like the one in the [demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/your-first-dashboard).
\ No newline at end of file
+6. With that config, your dashboard should look like the one in the [demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/your-first-dashboard).
```

Comment:
There should be no comma.

Reply:
Comma after the introductory phrase is better in my opinion.